.dockerignore

```
@@ -0,0 +1,4 @@
+.devspace
```

Missing `.git`

```suggestion
.git
.devspace
```

Take a look at this python specific dockerignore suggestion for more things you may want to include: https://github.com/GoogleCloudPlatform/getting-started-python/blob/main/optional-kubernetes-engine/.dockerignore